apps/dbmgr.py

```diff
@@ -103,10 +103,20 @@ def frames(self):
         """Overridden."""
         return (self.managerFrame,)
 
-    def sendDB(self):
-        """Emits a broadcastRequested signal with the database list."""
+    def sendDB(self, isAdded: bool, name: str):
+        """Emits a broadcastRequested signal with the database list and a logging message.
+        
+        Args:
+            isAdded: True if a database is added, otherwise False.
+            name: A file name of the updated database.
+        """
         msg = {"db": [db._asdict() for db in self.dbList]}
         self.broadcastRequested.emit("dbbus", json.dumps(msg))
+        self.broadcastRequested.emit(
+            "logbus",
+            f"Database {name} is added." if isAdded
+            else f"Database {name} is removed."
```

Comment:
Good, in this case, `if` clause and `else` clause are close enough, so we can see them together, immediately noticing that they form a ternary conditional expression.
👍